### DISCUSSION TITLE
Feature request: show popup near cursor

### DISCUSSION THREAD

**Discussion by benwaffle** [FelipeFTN/Emoji-Copy#64] [Ideas]
It would be really slick if the emoji selector showed up next to the cursor, like on macOS

**Comment by FelipeFTN** [comment]
Heyy, @benwaffle!! Hahaha yeah, that would be an amazing feature! :star_struck: 

**Thanks for the suggestion, I'm gonna keep this on track! :100: :rocket:**

**Reply by FelipeFTN** [reply]
Need to check if this is possible with [Gnome's GJS](https://gjs.guide/) But I really think there might be some way to achieve this behavior!
